components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1819,17 +1857,24 @@ subroutine output_cloud_optics_lw(state, tau)
       use ppgrid, only: pver
       use physics_types, only: physics_state
       use cam_history, only: outfld
+      use radconstants, only: rrtmg_lw_cloudsim_band
 
       type(physics_state), intent(in) :: state
       real(r8), intent(in), dimension(:,:,:) :: tau
+      real(r8), dimension(size(tau, 1), size(tau, 2)) :: emis
+      integer :: lw_band_index, ncol, nlay
+
+      ncol = state%ncol
+      nlay = pver
 
       ! Check values
       call assert_valid(tau(1:state%ncol,1:pver,1:nlwbands), 'cld_tau_lw')
 
       ! Output
-      call outfld('CLOUD_TAU_LW', &
-                  tau(1:state%ncol,1:pver,1:nlwbands), &
-                  state%ncol, state%lchnk)
+      call outfld('CLOUD_TAU_LW', tau(1:ncol,1:pver,1:nlwbands), ncol, state%lchnk)
+      lw_band_index = get_band_index_lw(10.5_r8, 'micron')
+      emis(1:ncol,:) = 1._r8 - exp(-tau(1:ncol,:,lw_band_index))
```

Comment:
This is probably a really dumb question, but as a sanity check: should this line be 1-exp(-tau/**cos(zenith)**)?

Reply:
No I don't think so. As this is longwave emissivity, the solar zenith angle doesn't factor in here at all.